### PR TITLE
html form

### PR DESCRIPTION
**Pull request by ghost**
https://yevheniia-biliaieva.github.io/layout_html-form/
https://yevheniia-biliaieva.github.io/layout_html-form/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,16 +2,137 @@
...
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <label class="label">
+          Surname:
+          <input
```
Comment (mykhalenych) [inline src/index.html:18]: Screenshot 2023-03-06 at 18 51 31

check all html tags

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 2]
```diff
@@ -2,16 +2,137 @@
...
+            type="text"
+            name="surname"
+            required
+            minlength="2"
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
```
Comment (Viktor-Kost) [inline src/index.html]: add an empty line between _multiline_ siblings
```suggestion
        </label>
        
        <label class="label">
```

File: src/index.html [hunk 3]
```diff
@@ -2,16 +2,137 @@
...
+            type="radio"
+            value="yes"
+            name="love-cats"
+          > Yes
+          <input
+            type="radio"
+            value="no"
+            name="love-cats"
+          > No
+        </label>
```
Comment (Viktor-Kost) [inline src/index.html]: each input should be wrapped into a label here
```suggestion
        <div class="label">
          Do you love cats?
          <label>
            <input
              type="radio"
              value="yes"
              name="love-cats"
            > 
            Yes
          </label>
         
          <label>
            <input
              type="radio"
              value="no"
              name="love-cats"
            > 
            No
          </label>
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
Nice!
Some improvements needed

File: src/index.html [hunk 4]
```diff
@@ -2,16 +2,152 @@
...
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+        <div class="label">
+          Do you love cats?
+          <label>
+            <input
+            type="radio"
+            value="yes"
+            name="love-cats"
+            >
```
Comment (IvanFesenko) [inline src/index.html]:
```suggestion
            <input
              type="radio"
              value="yes"
              name="love-cats"
            >
```

File: src/index.html [hunk 5]
```diff
@@ -2,16 +2,152 @@
...
+						Yes
+          </label>
+
+          <label>
+            <input
+            type="radio"
+            value="no"
+            name="love-cats"
+            >
```
Comment (IvanFesenko) [inline src/index.html]:
```suggestion
            <input
              type="radio"
              value="no"
              name="love-cats"
            >
```

**Review by IvanFesenko — APPROVED** [review]
Pay attention on indents

**Pull request closed without merging**